org.eclipse.xtext.core.idea.tests/.project

```diff
@@ -11,17 +11,12 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
```

Comment:
I just re-imported the project with the gradle import wizard and it kept stable, i.e. removing the builder here. Don't now why.